src/main/java/step3/model/LottoResult.java

```diff
@@ -0,0 +1,70 @@
+package step3.model;
+
+import step3.constant.ErrorMessageConstant;
+import step3.constant.LottoConstant;
+import step3.exception.LottoFormatException;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class LottoResult {
+    List<LottoNumber> lottoNumbers;
+
+    public LottoResult(List<LottoNumber> lottoNumbers) {
+        checkLottoOutOfSize((int) lottoNumbers.stream().distinct().count());
+        this.lottoNumbers = lottoNumbers.stream().distinct().sorted().collect(Collectors.toList());
+    }
+
+    public LottoResult(String[] lottoNumbersText) {
+        if (lottoNumbersText == null || lottoNumbersText.length == 0) {
+            throw new IllegalArgumentException(ErrorMessageConstant.EMPTY_TEXT);
+        }
+        this.lottoNumbers = getLottoNumbersFromTexts(lottoNumbersText);
+    }
+
+    private List<LottoNumber> getLottoNumbersFromTexts(String[] lottoNumbers) {
+        HashSet<LottoNumber> lottoSet = new HashSet<>();
```

Comment:
DIP 를 지키기 위해 변수 타입은 추상화된 클래스를 의존하는 것을 추천드릴게요!!
```suggestion
        Set<LottoNumber> lottoSet = new HashSet<>();
```

Reply:
앗 넵 감사합니다!